tests/unit_tests/test_ecephys.py

```diff
@@ -1,6 +1,14 @@
+from unittest import TestCase
+
+from uuid import uuid4
+from datetime import datetime
+
+import numpy as np
```

Comment:
```suggestion
import numpy as np
from unittest import TestCase
from uuid import uuid4
from datetime import datetime
```

Reply:
Yup, that's what I'm suggesting. All these are standard library imports for Python 3.9, no?

Reply:
Ah, numpy still isn't.